tests/components/image/test_init.py

```diff
@@ -260,6 +268,11 @@ async def test_fetch_image_url_success(
 
     client = await hass_client()
 
+    # Using HEAD
+    resp = await client.head("/api/image_proxy/image.test")
+    assert resp.status == HTTPStatus.OK
```

Comment:
It would be a good idea to check the expected metadata (Content Type, Content Length if applicable) are returned.

Reply:
Done.